test/assetcompute.test.js

```diff
@@ -48,6 +48,32 @@ describe( 'assetcompute.js tests', () => {
         assert.strictEqual(response.requestId, requestId);
         assert.strictEqual(response.journal, journal);
     });
+    it('should fail with asset compute /register hits 429', async function() {
```

Comment:
we dont have retry for this correct? can we mention it in the name ?
```suggestion
    it('should fail without retry when asset compute /register hits 429', async function() {
```